lib/run-script-pkg.js

```diff
@@ -31,7 +35,7 @@ const runScriptPkg = async options => {
   if (options.cmd)
     cmd = options.cmd
   else if (pkg.scripts && pkg.scripts[event])
-    cmd = pkg.scripts[event] + args.map(a => ` ${JSON.stringify(a)}`).join('')
+    cmd = sh`${unquoted(pkg.scripts[event])} ${args}`
```

Comment:
This is how Yarn does it:

https://github.com/yarnpkg/yarn/blob/3119382885ea373d3c13d6a846de743eca8c914b/src/cli/commands/run.js#L136

And how the original PR by the author of `puka` did it:

https://github.com/npm/cli/pull/52/files#diff-5fe49e56d3dc3f4bb6cead3ec153dbee2b55cc91f22bf991cac6546f6aa42623R172